types/react-virtualized/dist/es/Table.d.ts

```diff
@@ -441,7 +441,7 @@ export class Table extends PureComponent<TableProps> {
         onScroll: () => null;
         overscanRowCount: 10;
         rowRenderer: TableRowRenderer;
-        headerRowRenderer: TableHeaderRenderer;
+        headerRowRenderer: TableHeaderRowRenderer;
```

Comment:
Matches the definition in TableProps: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/35756/files#diff-4db0ed7777ff0246bfa9ee9878619ea6R233